tests/test_table.py

```diff
@@ -1620,6 +1712,15 @@ def test_update_ids_cache_bug(self):
         exp_index = {'x': 0, 'y': 1}
         self.assertEqual(obs._sample_index, exp_index)
 
+    def test_other_spmatrix_type(self):
+        # I dont actually remember what bug stemmed from this...
```

Comment:
It came up in two scenarios. I could dig through the code but not really excited about that as it would necessitate a bunch of parallel exploration in an interpreter. I could just delete the comment? :)